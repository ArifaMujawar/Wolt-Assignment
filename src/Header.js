import React from "react";

const Header = () => {
  return (
    <header>
      <h1 className="logo">Wolt</h1>
      <div className="searchBar">
        <input type="text" placeholder="search with street address"></input>
      </div>
    </header>
  );
};

export { Header as default };
