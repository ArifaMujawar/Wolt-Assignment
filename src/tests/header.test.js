import Header from '../Header.js'
import React from 'react'
import ReactShallowRenderer from 'react-test-renderer/shallow'

test('should render Header correctly',()=>{
  const renderer = new ReactShallowRenderer();
  renderer.render(<Header />);
  expect(renderer.getRenderOutput()).toMatchSnapshot();
});

