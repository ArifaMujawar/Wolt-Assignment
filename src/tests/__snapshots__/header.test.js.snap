// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<header>
  <h1
    className="logo"
  >
    Wolt
  </h1>
  <div
    className="searchBar"
  >
    <input
      placeholder="search with street address"
      type="text"
    />
  </div>
</header>
`;
